features/telemetry/adapters/collection/src/main/java/org/opennms/netmgt/telemetry/adapters/collection/AbstractPersistingAdapter.java

```diff
@@ -264,7 +263,7 @@ protected ScriptedCollectionSetBuilder getCollectionBuilder() throws Exception {
         if (builder == null) {
             // script didn't compile, set flag to false
             scriptCompiled.set(false);
-            throw new Exception(String.format("Error compiling script '%s'. See logs for details.", script));
```

Comment:
Is it worth logging this information?

Reply:
It's duplicated. The caller logs this in all cases. (i.e.: https://github.com/OpenNMS/opennms/blob/19d62fc7719042ad2c3ba6605486860ddf8ea17b/features/telemetry/adapters/nxos/src/main/java/org/opennms/netmgt/telemetry/adapters/nxos/NxosGpbAdapter.java#L135)